pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -501,6 +504,29 @@ func mutateContainerResource(c *corev1.Container, override *v1beta1.OverrideSpec
 			}
 		}
 	}
+
+	c.Args = append(c.Args, fmt.Sprintf("--v=%d", containerLogLevel(c.Name, override.LogLevels)))
+}
+
+// containerLogLevel will determine the log level value for any reconciler deployment container
+func containerLogLevel(containerName string, override []v1beta1.ContainerLogLevelOverride) int {
+	for _, logLevel := range override {
+		if logLevel.ContainerName == containerName {
+			return logLevel.LogLevel
+		}
+	}
+
+	return containerLogLevelDefault(containerName)
+}
+
+// containerLogLevelDefault will return the default log level value for any reconciler deployment container
+// when override is not specified
+func containerLogLevelDefault(containerName string) int {
+	// git-sync container's default is 5
+	if containerName == reconcilermanager.GitSync {
```

Comment:
nit: a switch statement would be prettier, but it's not a big deal.

Reply:
Agreed that a switch would make sense stylistically here, especially if we need to add additional cases

Reply:
changed to use switch